examples/config.sample

```diff
@@ -73,6 +74,12 @@
 # Default: 0 (disabled)
 #MaxLogFiles = 0
 
+# SnifferSysid
+# Sysid of sniffer. All traffic is forwarded to endpoints with this sys id.
+# Useful for debug porposes with a mavlink inspector on this endpont.
+#  Default: 0 (disabled)
```

Comment:
Please use the same style as the other descriptions. Like: "Forward all traffic to endpoints on which this MAVLink system ID is connected. This can be used to log or view all messages flowing though mavlink-router.
Default: 0 (disabled)"

Reply:
done